tests/crypto/test_ccm_mode/testcase.yaml

```diff
@@ -0,0 +1,3 @@
+tests:
+-   test:
+        tags: crypto aes ccm
```

Comment:
tags crypto aes ccm
test path crypto/ccm

We need to have the system pickup tags from the path, as there is info there that we don't need to repeat (it is an initial classification after all)